config/api.go

```diff
@@ -3,6 +3,7 @@ package config
 import (
```

Comment:
Not a fan of having all this defined in config. Can we move it to a new file `/lib/api.go`?